src/platform/platform_linux.cpp

```diff
@@ -42,23 +44,52 @@ mp::ProcessFactory* process_factory()
     }
     return static_process_factory.get();
 }
+
+QString get_driver_str()
+{
+    auto driver = qgetenv(mp::driver_env_var);
+    return driver.isEmpty() ? mp::Settings::instance().get(mp::driver_key) : driver.toLower();
+}
+
 } // namespace
 
 std::string mp::platform::default_server_address()
 {
     return {"unix:/run/multipass_socket"};
 }
 
-mp::VirtualMachineFactory::UPtr mp::platform::vm_backend(const mp::Path& data_dir)
+QString mp::platform::default_driver()
 {
-    auto driver = qgetenv("MULTIPASS_VM_DRIVER");
+    return QStringLiteral("qemu");
+}
 
-    if (driver.isEmpty() || driver == "QEMU")
+QString mp::platform::daemon_config_home() // temporary
+{
+    auto ret = QString{qgetenv("DAEMON_CONFIG_HOME")};
+    if (ret.isEmpty())
+        ret = QStringLiteral("/root/.config");
+
+    ret += QStringLiteral("/%1").arg(mp::daemon_name);
```

Comment:
Similar to the paths below?

```suggestion
    ret = QDir{ret}.absoluteFilePath(mp::daemon_name);
```

Reply:
OK, will do.